src/components/ContentWidgets/TabView/TabView.css

```diff
@@ -0,0 +1,92 @@
+.tabs-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    list-style-type: none;
+    padding: 1rem;
+    background-color: #420DAB;
+    color: #ffffff;
+}
+
+.tab:hover {
```

Comment:
`.tab` might be a pretty common class name, I would avoid using that. Maybe `tab-view` or something